docs2/site/docs/getting-started/metrics.md

```diff
@@ -1,6 +1,10 @@
 # Metrics
 
-Metrics are captured during execution.  This can help you determine performance issues within a resolver or validation.  Field metrics are captured using Field Middleware and the results are returned as a `PerfRecord` array on the `ExecutionResult`.  You can then generate [Apollo Tracing](https://github.com/apollographql/apollo-tracing) data with the `EnrichWithApolloTracing()` extension method.
+Metrics are captured during execution. This can help you determine performance issues within
```

Comment:
> Metrics are captured during execution.

Should this sentence be updated now that `EnableMetrics` defaults to `false`?

PR looks good otherwise.

Reply:
done